pupil-spa/src/app/services/azure-queue/azure-queue.service.ts

```diff
@@ -29,7 +29,7 @@ export class AzureQueueService {
    */
   public initQueueService(queueName: string, url: string, token: string, retryConfig): IQueueService {
     const service = this.queueStorage
-      .createQueueServiceWithSas(url.replace(queueName, ''), token)
+      .createQueueServiceWithSas(`${window.location.origin}/queue`, token)
```

Comment:
@danwilliams if I understand correctly this will be <app_url>/queue?
I am not sure I am following with this change but how will this address each queue and also how is this going to work with local development?

Reply:
As @GuyHarwood pointed out don't we need a fallback?